source/monitoring.txt

```diff
@@ -35,6 +35,104 @@ To learn more about logging in the connector, see the
 
 .. _kafka-monitoring-information:
 
+Use Cases
+---------
+
+This section describes use cases for monitoring MongoDB source and sink
+connectors.
+
+Sink Connector
+~~~~~~~~~~~~~~
+
+The following table describes some use cases for monitoring the MongoDB
+sink connector and the metrics the sink connector provides
+to satisfy those use cases:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Use Case
+     - Metrics to Use
+
+   * - You need to know the total number of records your connector
+       wrote to MongoDB.
+     - Use the ``records`` metric.
+
+   * - You need to know how long the interval of time is between
+       when a record arrives in a Kafka topic and when your connector writes
+       that record to MongoDB.
+     - Use the ``latest-kafka-time-difference-ms`` metric.
+
+   * - You need to know the total number of write errors your connector
+       encountered when attempting to write to MongoDB.
+     - Use the ``batch-writes-failed`` metric.
+
+   * - You need to know if your MongoDB performance is getting slower
+       over time.
+     - Use the ``getmore-commands-successful-duration-over-<number>-ms``
+       and ``getmore-commands-failed-duration-over-<number>-ms``
+       metrics. Track the change in these metrics over
+       time, and see if the values of the longer-duration metrics are increasing
+       at a faster rate than the values of the shorter-duration metrics.
```

Comment:
This last sentence was a bit confusing to me. I had to read over it a couple times to figure out what the "longer-duration metrics" and "shorter-duration-metrics" are. Is that referring to the amount you put in the <number> placeholder in the command?

Reply:
Good catch, this is definitely hard to parse. Had trouble thinking of this, but probably still more room for improvement. I'll think about this more and get something back to you.

Reply:
Split this into two sentences as follows:

Track the change in these metrics over time. See if the number of longer running commands is increasing at a faster rate than the number of the shorter running commands.